citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/AuthorityService.java

```diff
@@ -213,5 +223,14 @@ private AuthorityImporter getAuthorityImporter(String source) {
         }
         return null;
     }
+    
+    @Override
+    public String getGroupNameByGroupId(String citationGroupId) throws GroupDoesNotExistException {
```

Comment:
the group manager has a get group method, doesn't it? why don't you use that one?